packages/driver/cypress/integration/e2e/multi-domain/multi_domain_validation_specs.ts

```diff
@@ -0,0 +1,137 @@
+// @ts-ignore / session support is needed for visiting about:blank between tests
+describe('multi-domain', { experimentalSessionSupport: true, experimentalMultiDomain: true }, () => {
+  describe('successes', () => {
+    it('succeeds on a localhost domain name', () => {
+      cy.switchToDomain('localhost', () => undefined)
+    })
+
+    it('succeeds on an ip address', () => {
+      cy.switchToDomain('127.0.0.1', () => undefined)
+    })
+
+    // TODO: an ipv6 address passes validation but hangs elsewhere in the code, Come back later and fix ipv6
+    it.skip('succeeds on an ipv6 address', () => {
```

Comment:
Calling this out, ipv6 addresses hang for reasons outside of validation. I didn't dig in too far for fear of getting too sidetracked and missing my sprint goals